src/test/java/com/artipie/api/RepositoryVerticleTest.java

```diff
@@ -34,14 +39,33 @@ class RepositoryVerticleTest {
      */
     private static final BlockingStorage ASTO = new BlockingStorage(new InMemoryStorage());
 
+    /**
+     * Service host.
+     */
+    private static final String HOST = "localhost";
+
     /**
      * Service port.
      */
     private static int port;
 
+    /**
+     * Maximum awaiting time duration of port availability.
+     */
+    private static final Duration MAX_WAIT = Duration.ofMinutes(10);
```

Comment:
Is there any sense to use `Duration` rather than simple long?

Reply:
Ok, I'm not against using `Duration`, but do we really need to convert every time `duration` to long?
We can do it once:
```
private static final long MAX_WAIT = Duration.ofMinutes(10).toMillis();
```

Reply:
Fixed. Please review